main_test.go

```diff
@@ -173,3 +173,17 @@ func TestAddCommand(t *testing.T) {
 		result()
 	})
 }
+
+func TestRemoveCommand(t *testing.T) {
+	t.Run("it shows usage", func(tt *testing.T) {
+		args := []string{"ergo", "remove"}
+		os.Args = args
+
+		result := command()
+		if result == nil {
```

Comment:
You name the test _"it shows usage"_, but you only check if the command executed.

Reply:
added output check to see if usage was shown